src/app/plugins/plugin_colorcalib.cpp

```diff
@@ -21,13 +21,23 @@
 #include "plugin_colorcalib.h"
 #include <QStackedWidget>
 
+#define LUT_COPY_PLACEHOLDER    "(click copy to refresh)"
```

Comment:
Please do not use #defines unless absolutely necessary.
This functionality can be reproduced with the use of a static const string variable in the scope of the function using it.